20190618/clone_sqlflow_models.bashsample

```diff
@@ -0,0 +1 @@
+git clone git@github.com:<your Github account>/models.git
```

Comment:
Is there any way that can use inline code in the `.slide` file? maybe indentation?